### ISSUE TITLE
Update README for Mac Installation

### ISSUE THREAD

**Issue by buckyroberts** [thenewboston-developers/Account-Manager#292]
https://medium.com/flawless-app-stories/gyp-no-xcode-or-clt-version-detected-macos-catalina-anansewaa-38b536389e8d

**Comment by dahelg** [comment]
@buckyroberts I had this kind of error a couple of times in the past. (Currently I run on  macOS Catalina version 10.15.7.)

```
gyp: No Xcode or CLT version detected!
gyp ERR! configure error
```

The problem with the medium article is that the cli installation via `xcode-select --install` failes for some users with the message

> Can't install the software because it is not currently available from the Software Update server.

Because I was one of those lucky ones, I researched this and found that I have to download and install the Command Line Tools for Xcode in its latest version (currently 12) via [Apple's developer portal](https://developer.apple.com/download/more/?=command%20line%20tools).

If the issue shall be solved by updating the docs with a short description of your linked article and the trouble shooting for the error I described, I can do this.
